roles/lib_openshift/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 - name: lib_openshift ensure python-ruamel-yaml package is on target
   package:
-    name: python-ruamel-yaml
+    name: python2-ruamel-yaml
```

Comment:
This is a workaround until a proper python-ruamel-yaml image is available for CentOS.